src/test/regress/Pipfile

```diff
@@ -24,4 +24,4 @@ flake8 = "*"
 flake8-bugbear = "*"
 
 [requires]
-python_version = "3.9"
+python_version = "3"
```

Comment:
I think this change will lead to creating new images in [the-process](https://github.com/citusdata/the-process/) by using new pipfile contents.

Reply:
good thinking, let me check how this impacts `the-process` (CI Images).